src/stackedBars.ts

```diff
@@ -48,14 +52,55 @@ export function renderStackedBars(pareto: Pareto, settings: Settings) {
         .attr("shape-rendering", "crispEdges")
         .attr("stroke", (d) => (d.isMarked ? "#000" : "none"))
         .attr("stroke-width", (d) => (d.isMarked ? settings.style.selectionBox.strokeWidth : "0"));
-
+        
+    
+        
+     const range =  svgBoundingClientRect.width - resources.PADDINGRIGHT 
+     const percentageLabelPadding = svgBoundingClientRect.width /4;
+     let ppp = categoryAxisBandwidth.bandwidth()/4;
+   sel.append("line")
+        .style("stroke", "red")
+        .style("stroke-width", "6")
+        .style("stroke-dasharray", "8 8 ")
+        .attr("x1",  range)
+        .attr("y1", percentageAxis(80))
+        .attr("x2",0)
+        .attr("y2", percentageAxis(80))
+        .on("click",selectedBars)
+        // .on("mouseout", function () {

```

Comment:
Why is this outcommented? Is it not used anymore?

Reply:
I was waiting for the Tibco comment, now it's fixed.